setup.py

```diff
@@ -254,7 +259,9 @@ def build_extension(self, ext):
             hooks = getattr(ext, '_pre_build_hook', ())
             _run_pre_build_hooks(hooks, (self, ext))
 
-            old_build_ext.build_extension(self, ext)
+            # FIXME: this doesn't work, we need to call both the
+            # numpy.distutils and the pythran version of build_extension!
+            super(build_ext, self).build_extension(ext)
```

Comment:
This is the main issue I see. The multiple inherence is painful here, we have two incompatible `build_extension` methods. I'm not sure what the cleanest solution is.

@serge-sans-paille you mention the NumPy version at https://github.com/serge-sans-paille/pythran/blob/master/pythran/dist.py#L25, but I don't see how that works? You're invoking the `distutils` version of `build_extension` in the try-except block at the end of `PythranBuildExt.build_extension`.

Reply:
The pythran version is way simpler than the `numpy.distutils` version and doesn't do the actual build. So we could copy the contents of the `pythran` version into teh SciPy `setup.py` - but that doesn't feel very satisfactory.